arch/arm/core/aarch64/macro_priv.inc

```diff
@@ -122,6 +122,32 @@
 	eret
 .endm
 
+/**
+ * @brief Increment nested counter
+ *
+ * @return N/A
+ */
+
+.macro inc_nest_counter xreg0, xreg1, xreg2
```

Comment:
I don't understand purpose of xreg2, is that intentional?

Reply:
Not intentional. It is part of a following PR that yet posted based on this PR. I'll fix that.